Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Core/Classes/General/ToolTipValueManager.cs

```diff
@@ -6,8 +6,6 @@
  */
```

Comment:
Why is this class here, and not using the one from the Standard toolkit.
Also the Rest of the core should just be using the code from the Standard toolkit ?

Reply:
Removed class